ui/packages/atlasmap/src/UI/MappingTransformation.stories.tsx

```diff
@@ -63,5 +64,6 @@ export const nonRemovable = () => (
     noPaddings={boolean("Hide paddings", true)}
     onTransformationArgumentChange={action("onActionArgumentChange")}
     onTransformationChange={action("onActionChange")}
+    disableTransformation={false}
```

Comment:
Since there is no knob here I can’t really tell from the preview on Netlify, but shouldn’t also text inputs be disabled when this is true?

Reply:
yes - you're right.  Do you have an example for a storybook example with a knob?  Is it just an argument to the example?

Reply:
Check line #64